.github/workflows/update-manpage.yml

```diff
@@ -72,6 +73,7 @@ jobs:
           token: ${{ secrets.HOMEBREW_GITHUB_API_TOKEN }}
           branch: ${{ steps.update.outputs.branch }}
           force: true
+          origin_branch: "master"
```

Comment:
Would be good to investigate what other workflows could benefit from this.

Reply:
I'll take a look at this. It seems like the `sorbet`, `spdx` workflows haven't faced this problem because they aren't run on pushes to `master`.

Reply:
I _think_ I've seen this weirdness on both of them at some point (but I could be wrong)

Reply:
Seems like it might've happened in #9334 (this [commit](https://github.com/Homebrew/brew/commit/504ea3cf5b386196f959cb4f952042f25bd88e96)) at least. The `reset --hard` was removed in #8468.